### ISSUE TITLE
XML escaping removed when using "Copy response to clipboard"

### ISSUE THREAD

**Issue by holandaleo** [a85/POSTMan-Chrome-Extension#1219]
Postman is interpreting the XML escaping when using the "Copy response to clipboard" button.

Use this block as response body:

``` xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<body>
    <test>I have properly handled &amp; escaped my xml</test>
    <test>This is a less than: &lt</test>
    <test>This is a greater than: &gt</test>
</body>
```

After hitting the "Copy response to clipboard" and pasting the clipboard contents to a text editor, the contents I got:

``` xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<body>
    <test>I have properly handled & escaped my xml</test>
    <test>This is a less than: <</test>
    <test>This is a greater than: ></test>
</body>
```

The workaround is to select "raw" and then manually click the response panel and select all (CTRL+A) then copy (CTRL+C). This way, the escaping is not lost.

Environment:
Version 3.0.2 (packaged)
OS: win
Architecture: x86-32
Native client architecture: x86-64

**Comment by abhijitkane** [comment]
Thanks for reporting this - we'll push a fix for this in v3.0.3 of the app.

**Comment by abhijitkane** [comment]
@holandaleo - Can you confirm this fix in v3.0.4?

**Comment by holandaleo** [comment]
Yeah. Just got v3.0.4 now. Thx.
